collect_app/src/main/java/org/odk/collect/android/database/entities/DatabaseEntitiesRepository.kt

```diff
@@ -399,29 +397,28 @@ class DatabaseEntitiesRepository(context: Context, dbPath: String) : EntitiesRep
         }
     }
 
+    private fun addPropertiesToContentValues(contentValues: ContentValues, entity: Entity) {
+        entity.properties.forEach { (name, value) ->
+            contentValues.put("\"${EntitiesTable.getPropertyColumn(name)}\"", value)
```

Comment:
Couldn't this be just:
`contentValues.put(EntitiesTable.getPropertyColumn(name), value)`

Reply:
You'll see that `#save supports properties with dots and dashes when saving new entities and updating existing ones` fails if you change to that. We need to quote the column property column names as they might contain special characters that SQL queries don't support.